classes/book.rb

```diff
@@ -8,4 +8,9 @@ def initialize(publish_date, publisher, cover_state)
     @publiser = publisher
     @cover_state = cover_state
   end
+
+  def can_be_archived?
+    true if can_be_archived? || @cover_state == 'bad'
```

Comment:
In this method you are trying to _`return true if parent's method returns true OR if cover_state equals to "bad"`_

- And the way you get parents method is by using the `super` method
- E.g: `super || @cover_state == bad`

Reply:
Thanks, for referring 

I fixed the issue.